.github/workflows/pkgdown.yaml

```
@@ -57,6 +57,11 @@ jobs:
         shell: Rscript {0}
         run: |
           pak::pkg_install("pkgdown", dependencies = TRUE)
+          pak::pkg_install("dplyr", dependencies = TRUE)
+          pak::pkg_install("readr", dependencies = TRUE)
+          pak::pkg_install("stringr", dependencies = TRUE)
+          pak::pkg_install("purrr", dependencies = TRUE)
+          pak::pkg_install("glue", dependencies = TRUE)
```

This way we can freely add/remove pkgs from the `vignettes/` folder

```suggestion
          pkgs <- unique(renv::dependencies("vignettes")$Package)
          lapply(pkgs, function(pkg) {
            if (! (pkg %in% rownames(installed.packages()))) {
              pak::pkg_install(pkg, upgrade = TRUE, dependencies = TRUE)
            }
          })
```